CRM/Contribute/BAO/Contribution.php

```diff
@@ -132,6 +132,7 @@ public static function add(&$params, $ids = array()) {
     else {
       // @todo put a deprecated here - this should be done in the form layer.
```

Comment:
@eileenmcnaughton if this passes i think we should remove this to do

Reply:
lol